packages/frontend/core/src/pages/workspace/all-page/all-page-header.tsx

```diff
@@ -59,8 +59,12 @@ export const AllPageHeader = ({
               styles.headerCreateNewButton,
               !showCreateNew && styles.headerCreateNewButtonHidden
             )}
-            onCreateEdgeless={createEdgeless}
-            onCreatePage={createPage}
+            onCreateEdgeless={e =>
+              createEdgeless(e?.metaKey || e?.ctrlKey ? 'new-tab' : true)
```

Comment:
a todo is really needed here. such logic works but too raw.

Reply:
what about a `hasModifiers` check for metaKey/ctrlKey?

Reply:
day after again say